draft-ietf-ppm-dap-taskprov.md

```diff
@@ -467,24 +467,34 @@ which in turn is presumed to be configured out-of-band.
 The process of provisioning a task begins when the Author disseminates the task
 configuration to the Collector and each of the Clients. When a Client issues an
 upload request to the Leader (as described in {{Section 4.5 of !DAP}}), it
-includes in an HTTP header the task configuration it used to generate the
-report. We refer to this process as "task advertisement". Before consuming the
-report, the Leader parses the configuration and decides whether to opt-in; if
-not, the task's execution halts.
+includes in an HTTP Structured Header {{!RFC9651}} the task configuration it
+used to generate the report. We refer to this process as "task advertisement".
+Before consuming the report, the Leader parses the configuration and decides
+whether to opt-in; if not, the task's execution halts.
 
 Otherwise, if the Leader does opt-in, it advertises the task to the Helper
 during the aggregation interaction ({{Section 4.6 of !DAP}}). In particular, it
 includes the task configuration in an HTTP header of each aggregation job
 request for that task. Before proceeding, the Helper must first parse the
 configuration and decide whether to opt-in; if not, the task's execution halts.
 
-## Task Advertisement {#task-advertisement}
+## DAP-Taskprov Structured Header {#task-advertisement}
 
-To advertise a task to its peer, a protocol participant includes the header
-"DAP-Taskprov" with an HTTP request incident to the task execution. The value
-is the `TaskConfig` structure defined {{task-encoding}}, expanded into its
-URL-safe, unpadded Base 64 representation as specified in {{Sections 5 and 3.2
-of !RFC4648}}.
+The DAP-Taskprov HTTP header field is used to advertise a task.
+
+DAP-Taskprov is an Item Structured Header Field {{!RFC9651}}. Its value MUST be
+a Byte Sequence ({{Section 3.3.5 of !RFC9651}}).
```

Comment:
You need to say a) what to do when it isn't an Item (that is, badly formed) or a Byte Sequence (ignore it is my suggestion, particularly given the question you ask above) and b) what to do with parameters (again, ignoring unknown parameters is recommended).